tests/python/rest_api/test_tasks.py

```diff
@@ -2733,3 +2733,49 @@ def test_can_export_and_import_skeleton_tracks_in_coco_format(self):
                 for tes in te.shapes
             ]
         )
+
+
+@pytest.mark.usefixtures("restore_db_per_function")
+class TestGetTaskAnnotations:
+    @pytest.mark.parametrize("mode", ["annotation", "interpolation"])
+    def test_datumaro_export_without_annotations_includes_image_info(self, admin_user, tasks, mode):
+        task = next(t for t in tasks if t["size"] and t["mode"] == mode)
+
+        with make_api_client(admin_user) as api_client:
+            response = export_dataset(
+                api_client.tasks_api.retrieve_annotations_endpoint,
+                id=task["id"],
+                format="Datumaro 1.0",
+            )
+            assert response.status == HTTPStatus.OK
+
+        with zipfile.ZipFile(io.BytesIO(response.data)) as zip_file:
+            annotations = json.loads(zip_file.read("annotations/default.json"))
+
+        assert annotations["items"]
+        for item in annotations["items"]:
+            assert item["media"]["path"]
+            assert item["image"]["path"]
```

Comment:
What is the difference between a media path and an image path in Datumaro format?

Reply:
The meaning is the same, but each possible media type is saved in a different field. If the type info is provided, the image field is used. If no media type info is available, the generic media type (and field) is used.

Reply:
Probably, the reasons for both fields to exist simultaneously are historical. Media path is just newer, and the image path is duplicated for backward compatibility.